File: qasdk/src/main/java/co/astrnt/qasdk/dao/CandidateApiDao.java
package co.astrnt.qasdk.dao;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

/**
 * Created by deni rohimat on 06/04/18.
 */
public class CandidateApiDao extends RealmObject {

    @PrimaryKey
    private long id;
    private String email;
    private String fullname;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getFullname() {
        return fullname;
    }

    public void setFullname(String fullname) {
        this.fullname = fullname;
    }
}

File: qasdk/src/main/java/co/astrnt/qasdk/dao/CandidateApiDao.kt
package co.astrnt.qasdk.dao

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class CandidateApiDao : RealmObject() {
    @PrimaryKey
    var id: Long = 0
    var email: String? = null
    var fullname: String? = null
}